main/get_feature_set.py

```diff
@@ -0,0 +1,139 @@
+def get_feature_set(dat):
+  if dat == "G":
+     feature_set = [
+                 ["rounds", "shepherd_time", "review_time"], # logs
```

Comment:
does it make sense to reduce duplication? this would help the reader see what subsets of features are being used across data sets.

Reply:
lemme try


> Success! Output: https://docs.google.com/document/d/1m-QS465k0K0sK_zxQ_6m1Z-HQrsUFepxNrXSCFrg-gY/edit

oh! the results look reasonable